### ISSUE TITLE
Is this still current?

### ISSUE THREAD

**Issue by surfjedi** [KinveyApps/StatusShare-iOS#8]
I have just tried to download and build this, but getting errors and it looks like this repo hasn't bee updated for years?

Is there a current working version and/or a swift version?

**Comment by tejasranade** [comment]
@surfjedi Yes, this is updated periodically (it was last updated in March).
We're providing Swift samples for the new 3.0 version of the Kinvey iOS SDK (currently in Beta). Please refer to this [IOS-Starter](https://github.com/KinveyApps/IOS-Starter) to get started.
